install.sh

```diff
@@ -218,7 +218,7 @@ pip2 install --user neovim 2>&1
 cd ~/.vim/bundle/tern_for_vim && npm install 2>&1;
 
 fancy_echo "Changing system Bash to newer Brew Bash"
-fancy_echo "If this fails, please do `chsh -s /usrl/local/bin/bash` manually"
+fancy_echo "If this fails, please do `chsh -s /usr/local/bin/bash` manually"
```

Comment:
I'm not sure whether we want to change this `usr` to `usrl`, because the automated script also says `usr`